src/app/ui/AppProvider/ToastProvider.tsx

```diff
@@ -3,7 +3,17 @@ import { FC, PropsWithChildren } from 'react';
 import { styled } from '@tamagui/core';
 import { ToastProvider as RNTNToastProvider } from 'react-native-toast-notifications';
 
-import { Box, Text } from '@app/shared/ui';
+import { colors } from '@app/shared/lib';
+import {
+  Box,
+  Text,
+  MaterialAlertOctagon,
+  OcticonsCircleCheckFill,
+} from '@app/shared/ui';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+import { Platform } from 'react-native';
```

Comment:
Did you set up auto-format on save? It seems like the editor should've automatically fixed the import order, but it didn't for some reason.
According to the ESLint rules, react-native should be on top and react-native-safe-area-context should be before react-native-toast-notifications

Reply:
+1

If you use VS Code, then you should have the ESLint extension enabled, and configured something like this in your VS Code settings:
```json
  "eslint.format.enable": true,
  "editor.formatOnSave": true,
```

Then this code should be flagged with eslint errors:

<img width="763" alt="Screenshot 2024-02-20 at 6 29 43 PM" src="https://github.com/ChildMindInstitute/mindlogger-app-refactor/assets/1282772/b6ba74fe-1475-405a-a2b5-e66c826e8ebf">


and automatically fixed to this save:

<img width="762" alt="Screenshot 2024-02-20 at 6 29 49 PM" src="https://github.com/ChildMindInstitute/mindlogger-app-refactor/assets/1282772/aba3854c-6164-49e8-ac09-88c645db5d6c">